lib/elements/autocomplete.js

```diff
@@ -97,21 +97,30 @@ class AutocompletePrompt extends Prompt {
   _(c, key) {
     let s1 = this.input.slice(0, this.cursor);
     let s2 = this.input.slice(this.cursor);
-    this.cursor = this.cursor+1; 
     this.input = `${s1}${c}${s2}`;
+    this.cursor = s1.length+1;
```

Comment:
Not sure I know why but it doesn't work when you just increment cursor.
Without this change, if you type `test`, then place your cursor after the `e` (`te|st`), then press backspace twice to delete `te`, then press delete twice to delete `st` and then if you type something the first letter will be added after the cursor position (e.g. if you write `test` it'll print `estt`)